modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java

```diff
@@ -189,14 +189,10 @@ public void setExtendedProperties(
 
 			ObjectRelationship objectRelationship =
 				_objectRelationshipLocalService.
-					fetchObjectRelationshipByObjectDefinitionId1(
+					getObjectRelationshipByObjectDefinitionId(
```

Comment:
Be careful! The previous method is using the objectDefinitionId1 and the new one is using objectDefinitionId1 or objectDefinitionId2, is that the expected behaviour?

Reply:
Make sense @ccorreagg!! I suppose that it was using the objectDefintion1, my bad